app/gui/src/presenter/graph/state.rs

```diff
@@ -444,13 +444,13 @@ impl<'a> ControllerChange<'a> {
         };
         let mut nodes = self.nodes.borrow_mut();
         let displayed = nodes.get_mut_or_create(ast_id);
-        tracing::debug!(
-            "Setting node expression from controller: {} -> {}",
-            displayed.expression,
-            new_displayed_expr
-        );
 
         if displayed.expression != new_displayed_expr {
+            tracing::debug!(
```

Comment:
don't use `tracing::debug` use `debug` instead. Its exported in prelude. Apply everywhere, please.

Reply:
Done